osf/models/schema_response_block.py

```diff
@@ -0,0 +1,22 @@
+from django.db import models
+from osf.models.base import BaseModel, ObjectIDMixin
+from osf.utils.datetime_aware_jsonfield import DateTimeAwareJSONField
+
+
+class SchemaResponseBlock(ObjectIDMixin, BaseModel):
+
+    # The SchemaResponses version where this response originated
+    source_revision = models.ForeignKey(
```

Comment:
I think this should just be called `source_schema_response`, just to be super clear.